src/workflows/orders.ts

```diff
@@ -61,6 +73,58 @@ export async function createOrderWorkflow(
   };
 }
 
+export async function createOrderWorkflowWithSigner({
+  l1Signer,
+  l2Signer,
+  request,
+  ordersApi,
```

Comment:
now most of the workflow function interfaces are using flat params instead an object interface(i am a fan), but this could be very inconsistent

Reply:
we discussed about it and will/need to change the other function to follow this standard.